docker-compose.yml

```diff
@@ -6,12 +6,39 @@ services:
       target: build
       args:
         nodeEnv: development
-    ports:
-      - "8686:8686"
-    depends_on:
-      - dgraph
     environment:
       DGRAPH_URL: http://dgraph:8080
       INTEGRATION_TEST: "true"
-  dgraph:
-    image: "dgraph/standalone:v21.03.2"
```

Comment:
Why are changing the docker to invidisual alpha and zero?

Reply:
Just used the https://github.com/dgraph-io/dgraph/blob/main/contrib/local-test/docker-compose.yml
as a reference. dgraph-lambda has not been updated in a while and I took the reference from dgraph repository.